.github/workflows/tests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ['8.0']
+                php: ['8.1']
```

Comment:
please revert to see if it cause the error, it should be ok with 8.0 as we have other downgrade 8.1 that working ok

Reply:
It's reverted. Now you can see the error.

FYI I run my PRs on my own repo before opening them here.

The 3 other downgrade 8.1 rectors you mentionned are syntactically valid before PHP 8.1, even if they would crash at runtime, that's why we didn't see this error before.

Reply:
/cc @ondrejmirtes probably related with RichParser?

Reply:
By tuning PHPStan's configuration it now works with PHP 8.1.